content/4-component-composition/1-props/lit/user-profile.js

```diff
@@ -0,0 +1,26 @@
+import { LitElement, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+@customElement('user-profile')
+export class UserProfile extends LitElement {
+	@property()
```

Comment:
To match with other props style, the type string is missing

Reply:
Sounds good! I could add a comment like: `@property({ type: String }) // Or just "@property()"` to show it is treated as a string by default.